uberon_edit.obo

```diff
@@ -193951,40 +193951,84 @@ created_by: http://orcid.org/0000-0001-9897-3238
-synonym: "hepatic acinus pericentral zone" EXACT [PMID:28126520]
-synonym: "hepatic acinus zone 3" EXACT [PMID:28126520]
+name: hepatic acinus zone 3
+def: "The region of the hepatic acinus immediately surrounding the central veins. This region has the lowest oxygen concentration in the acinus. Hepatocytes in this region are the primary location for the biotransformation of drugs." [PMID:29763190]
+synonym: "liver lobule centrilobular region" RELATED [] {RELATED="PMID:28126520"}
 is_a: UBERON:0000063 ! organ subunit
-relationship: adjacent_to UBERON:0006841 ! central vein of liver
+is_a: UBERON:8400005 ! metabolic zone of liver
 relationship: dc-creator http://orcid.org/0000-0003-2034-601X
 relationship: part_of UBERON:0001172 ! hepatic acinus
 property_value: http://purl.org/dc/elements/1.1/date 2020-09-08T15:05:03Z xsd:dateTime
 
 [Term]
 id: UBERON:8400003
-name: hepatic acinus intermediary zone
-def: "The region of the hepatic acinus in between the periportal zone and the perivenous zone. Blood in this region has an intermediate oxygen concentration relative to the other two zones of the hepatic acinus." [PMID:28126520]
-synonym: "hepatic acinus zone 2" EXACT [PMID:28126520]
+name: hepatic acinus zone 2
+def: "The region of the hepatic acinus in between zones 1 and 3. Blood in this region has an intermediate oxygen concentration relative to the other two zones of the hepatic acinus. Hepatocytes in this region have mixed functionality in comparison with those in zones 1 and 3." [PMID:29763190]
+synonym: "liver lobule midzonal region" RELATED [PMID:28126520]
 is_a: UBERON:0000063 ! organ subunit
-relationship: adjacent_to UBERON:8400001 ! hepatic acinus periportal zone
-relationship: adjacent_to UBERON:8400002 ! hepatic acinus perivenous zone
+is_a: UBERON:8400005 ! metabolic zone of liver
+relationship: adjacent_to UBERON:8400001 ! hepatic acinus zone 1
+relationship: adjacent_to UBERON:8400002 ! hepatic acinus zone 3
 relationship: dc-creator http://orcid.org/0000-0003-2034-601X
 relationship: part_of UBERON:0001172 ! hepatic acinus
 property_value: http://purl.org/dc/elements/1.1/date 2020-09-08T15:15:57Z xsd:dateTime
 
+[Term]
+id: UBERON:8400005
+name: metabolic zone of liver
+def: "Any of the metabolic zones found in the hepatic acinus. Liver tissue is heterogenous at the level of morphometry and histochemistry. This heterogeneity is linked to the position of a cell within the functional unit of the tissue, which, in turn, is related to the blood supply: Cells located in the upstream zone differ from those in the downstream zone with respect to subcellular structures, key enzymes, translocators, and receptors and, therefore, have different metabolic capacities. This is the basis for “metabolic zonation,” a concept first proposed for carbohydrate metabolism and later expanded to include amino acid and ammonia metabolism, xenobiotic metabolism, protective metabolism, and formation of plasma proteins." [PMID:8839925]
```

Comment:
Change smart quotes to regular double quotes to fix Travis job,

Reply:
fixed in bec1dd0ece8421de59f5acb5a08b048317bbadf7